Day04(1)/Day04(1)/Day04(1)/Library.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Day04;
+
+
+
+public class Library
+{
+    private List<Book> books = new List<Book>();
+
+    public Library()
+    {
+        books.Add(new Book("wigni1", "giorgi1", 1942));
+        books.Add(new Book("wigni2", "giorgi2", 1945));
+        books.Add(new Book("wigni3", "giorgi3", 1948));
+
+    }
+
+    public void printBook()
+    {
+        int i = 1;
```

Comment:
თუ ინდექსი გჭირდება, foreach-ის მაგივრად for-ით უფრო მარტივად და ზედმეტი ცვლადის გარეშე იზამ